modules/nodes-descheduler-profiles.adoc

```diff
@@ -90,9 +90,32 @@ Do not enable `CompactAndScale` with any of the following profiles: `LifecycleAn
 
 endif::nodes[]
 ifdef::virt[]
-Use the `LongLifecycle` profile to enable the descheduler on a virtual machine. This is the only descheduler profile currently available for {VirtProductName}. To ensure proper scheduling, create VMs with CPU and memory requests for the expected load.
+Use the `LongLifecycle` or `DevKubeVirtRelieveAndMigrate` profile to enable the descheduler on a virtual machine. To ensure proper scheduling, create virtual machines with CPU and memory requests based on the expected load.
```

Comment:
I'd prefer to mention `DevKubeVirtRelieveAndMigrate` before `LongLifecycle` since we want to have customers moving there.
I'd also mention that `DevKubeVirtRelieveAndMigrate` is an enhanced version of `LongLifecycle` with more features.